src/main/java/io/horizen/lambo/car/api/CarApi.java

```diff
@@ -54,9 +57,12 @@
 public class CarApi extends ApplicationApiGroup {
 
     private final SidechainTransactionsCompanion sidechainTransactionsCompanion;
+    private CarInfoDBService carInfoDBService;
```

Comment:
CarInfoDBService must be controlled only by CarRegistryApplicationState class. Other way it is possible to change the state of CarInfoDBService directly in `CarApi`.
I guess that it is done in this way, because there is no way to get access to the application state instance  inside the CarApi, because there is no interface method inside the `SidechainNodeView`, right?

Reply:
PS: in the SDK we are going to add a possibility to get ApplicationState and ApplicationWallet instances from the SidechainNodeView

Reply:
yes the reason was that one!
wating for the method will be available in SDK  I could otherwise inject ApplicationState in the constructor
what do you think?

Reply:
Let's keep it as is and post the issue to the lambo repo.
After the changes in the SDK we will change here as well.
Note: it's important, that `SidechainNodeView` should provide the access to the "read-only" part of the `CarRegistryApplicationState`, otherwise we will have the same issue as now.

Reply:
ok! I add the issue